haskell/test/Test/Ligo/BaseDAO/Proposal/Tokens.hs

```diff
@@ -38,11 +40,39 @@ freezeTokens originateFn = do
       , tiTxs = [FA2.TransferDestination { tdTo = unTAddress dodDao, tdTokenId = FA2.theTokenId, tdAmount = 10 }]
       }]])
 
+checkFreezeHistoryTracking
+  :: (MonadEmulated caps base m, HasCallStack)
+  => (ConfigDesc Config -> OriginateFn m)
+  -> GetFreezeHistoryFn m
+  -> m ()
+checkFreezeHistoryTracking originateFn getFreezeHistory = do
+  DaoOriginateData{..} <- originateFn testConfig defaultQuorumThreshold
+  let frozen_scale_value = 2
+  let frozen_extra_value = 10
+  let proposalMeta1 = ""
+  let proposalSize1 = fromIntegral . BS.length $ proposalMeta1
+  let requiredFrozen = proposalSize1 * frozen_scale_value + frozen_extra_value
+
+  withSender dodOwner1 $ call dodDao (Call @"Freeze") (#amount .! requiredFrozen)
+  advanceLevel 11
```

Comment:
```suggestion
  advanceLevel dodPeriod
```

Reply:
Done. Had to fix in a lot of places, I think these changes got lost during a rebase.

Reply:
IIRC it was mostly the opposite, when you opened this PR we hadn't added `dodPeriod` yet and there was no conflict during rebase, so they just didn't pop out.

I had noticed before, but forgot to comment about them :slightly_smiling_face: